.github/workflows/build.yml

```diff
@@ -38,6 +38,12 @@ jobs:
           # `service mysql restart` fails.
           - {os: ubuntu-20.04, ruby: 2.4, db: mysql80, allow-failure: true}
           - {os: ubuntu-18.04, ruby: 'head', db: '', allow-failure: true}
+          # db: the DB's brew package name in macOS case.
+          # Allow failure due to the following test failures.
+          # https://github.com/brianmario/mysql2/issues/965
+          - {os: macos-latest, ruby: 2.4, db: 'mariadb@10.4', allow-failure: true}
+          # Allow failure due to Mysql2::Error: Unknown system variable 'session_track_system_variables'.
+          - {os: macos-latest, ruby: 2.4, db: 'mysql@5.6', allow-failure: true}
```

Comment:
While there was only one case mysql 5.6 in macOS, I added one more diverse case mariadb 10.4 that looks the latest version on the macOS environment, seeing `brew search mariadb`. It is to test the macOS case and the testing logic on GitHub Actions with diverse cases. It seems that macOS is slower than Ubuntu in GitHub Actions. But I think having the 2 macOS cases is still okay.

Reply:
I think MySQL 5.6 is pretty old at this point. MySQL 8.0 is the way to go.

Reply:
@sodabrew OK I will change it to MySQL 8.0. By the way, I noticed that the brew package `mysql` without `@X.Y` is MYSQL 8.0, and the brew package `mariadb` is 10.5. These are the latest version on the macOS environment. I will change MariaDB to MariaDB 10.5 too.

```
mysql # => This is 8.0
mysql@5.6
mysql@5.7
mariadb => This is 10.5
mariadb@10.1
mariadb@10.2
mariadb@10.3
mariadb@10.4
```